Makefile

```diff
@@ -0,0 +1,85 @@
+.PHONY: build-garage-ci build-garage-headless build-garage-nvidia \
+	run-garage-ci run-garage-headless run-garage-nvidia
+
+SHELL = bash
```

Comment:
i don't think you have a guarantee bash is installed. is there a reason you need bash? i've basically never seen this in a Makefile before.

Reply:
bash is required to obtain the CodeCov arguments and pass them to the CI container:
```
run-ci:                                                                         
         docker run \                                                            
                 "$$(bash <(curl -s https://codecov.io/env))" \ 
                 ...
```
If the SHELL is not set as bash, even if the bash command is explicitly called in the target, the bourne shell is used. I can set the shell to be bash only in the run-ci target, for example:
```
run-ci: SHELL := /usr/bin/env bash
run-ci:                                                                         
         docker run \                                                            
                 "$$(bash <(curl -s https://codecov.io/env))" \ 
                 ...
```

Reply:
I did that and worked, so let me know if this is okay with you, otherwise I can explore other options without using bash such as using a temporary file to save the script.

Reply:
i like this solution. i think it might be more clear if you can put the results of the curl into a descriptive variable first (e.g. CI_ENV).

otherwise it would be hard for someone to figure out why we have a random curl call in our Docker run command.

Reply:
Done